### PR TITLE
2.7.4

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by DavidKang** [comment]
LGTM

File: src/api/test/functional/attributes_test.rb [hunk 1]
```diff
@@ -260,6 +260,32 @@ def test_invalid_get
...
+    # same with attribute parameter
+    post "/source/home:tom/_attribute/OBS:Issues", data
+    assert_response 403
+  end
+
+  def test_attrib_delete_permissions
+    # create an admin only attribute
+    login_king
+    data = "<attributes><attribute namespace='OBS' name='VeryImportantProject'/></attributes>"
+    post "/source/home:tom/_attribute", data
```
Comment (adrianschroeter) [inline src/api/test/functional/attributes_test.rb]: assert_xml :success

is missing here

**Comment by adrianschroeter** [comment]
looks good (except the one missing assert of the post operation, should get fixed in master at least)

File: src/api/test/functional/attributes_test.rb (hunk 1 above)
Comment (bgeuken) [inline src/api/test/functional/attributes_test.rb]: oh, good catch!

Comment (bgeuken) [inline src/api/test/functional/attributes_test.rb]: I forgot the release notes. Additional PR with the missing assert -> #2825

**Pull request merged**